practice2/AboutMePage.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<head>
+    <link rel="stylesheet" type="text/css" href="AboutMePageStyle.css">
+     <title>About Joseph Kennedy</title> 
+</head>
+<body>
+    <h1>About Joseph Kennedy</h1>
+    <h2>“It’s the job that’s never started as takes longest to finish.” -J.R.R Tolkien</h2>
```

Comment:
this is not rendering properly in Firefox ...
![Screenshot from 2019-03-10 19-18-01](https://user-images.githubusercontent.com/5642399/54093020-4af97400-4369-11e9-8de5-331938a61722.png)

one interesting challenge of web development is that different browsers can do different things